Zend/zend_API.c

```diff
@@ -539,7 +555,7 @@ static const char *zend_parse_arg_impl(int arg_num, zval *arg, va_list *va, cons
 					break;
 				} else {
 					if (is_callable_error) {
-						*severity = E_WARNING;
+						*severity = strict ? (E_EXCEPTION | E_ERROR) : E_WARNING;
```

Comment:
I didn't get. Now in "weak" mode we produce WARNING and return from function NULL (as we did it before). In "strict" mode we throw exception. What is wrong?